metrics/web/test/debug_menu/presentation/state/debug_menu_notifier_test.dart

```diff
@@ -0,0 +1,419 @@
+      () {
+        notifier.initializeDefaults();
+
+        final initialViewModel = notifier.fpsMonitorLocalConfigViewModel;
+
+        final isFpsMonitorEnabled =
+            notifier.fpsMonitorLocalConfigViewModel.isEnabled;
+        final expectedParam = LocalConfigParam(
+          isFpsMonitorEnabled: !isFpsMonitorEnabled,
+        );
+
+        when(updateUseCase(expectedParam)).thenAnswer(
+          (_) => Future.value(
+            LocalConfig(isFpsMonitorEnabled: !isFpsMonitorEnabled),
+          ),
+        );
+
+        notifier.toggleFpsMonitor();
+
+        expect(
+          notifier.fpsMonitorLocalConfigViewModel,
+          equals(initialViewModel),
+        );
+      },
+    );
+
+    test(
+      ".toggleFpsMonitor() sets the .localConfigUpdatingErrorMessage if an error occured when updating the local config persistent store",
+      () {
+        notifier.initializeDefaults();
+
+        final isFpsMonitorEnabled =
+            notifier.fpsMonitorLocalConfigViewModel.isEnabled;
+        final expectedParam = LocalConfigParam(
+          isFpsMonitorEnabled: !isFpsMonitorEnabled,
+        );
+
+        when(updateUseCase(expectedParam)).thenAnswer(
+          (_) => throw const PersistentStoreException(),
+        );
+
+        notifier.toggleFpsMonitor();
+
+        expect(
+          notifier.localConfigUpdatingErrorMessage,
+          isNotNull,
+        );
+      },
+    );
+
+    test(
+      ".toggleFpsMonitor() resets the .localConfigUpdatingErrorMessage",
+      () {
+        notifier.initializeDefaults();
+
+        final isFpsMonitorEnabled =
+            notifier.fpsMonitorLocalConfigViewModel.isEnabled;
+        final expectedParam = LocalConfigParam(
+          isFpsMonitorEnabled: !isFpsMonitorEnabled,
+        );
+
+        when(updateUseCase(expectedParam)).thenAnswer(
+          (_) => throw const PersistentStoreException(),
+        );
+
+        notifier.toggleFpsMonitor();
+
+        reset(updateUseCase);
+        when(updateUseCase(expectedParam)).thenAnswer(
+          (_) => Future.value(
+            LocalConfig(isFpsMonitorEnabled: !isFpsMonitorEnabled),
+          ),
+        );
+
+        notifier.toggleFpsMonitor();
+
+        expect(
+          notifier.localConfigUpdatingErrorMessage,
+          isNull,
+        );
+      },
+    );
+
+    test(
+      ".dispose() calls the close local config storage use case",
+      () {
+        notifier.dispose();
+
+        verify(closeUseCase()).called(1);
+      },
+    );
+  });
+}
+
+class _OpenLocalConfigStorageUseCase extends Mock
+    implements OpenLocalConfigStorageUseCase {}
+
+class _ReadLocalConfigUseCase extends Mock implements ReadLocalConfigUseCase {}
+
+class _UpdateLocalConfigUseCase extends Mock
+    implements UpdateLocalConfigUseCase {}
+
+class _CloseLocalConfigStorageUseCase extends Mock
+    implements CloseLocalConfigStorageUseCase {}
```

Comment:
```suggestion
class _OpenLocalConfigStorageUseCaseMock extends Mock
    implements OpenLocalConfigStorageUseCase {}

class _ReadLocalConfigUseCaseMock extends Mock implements ReadLocalConfigUseCase {}

class _UpdateLocalConfigUseCaseMock extends Mock
    implements UpdateLocalConfigUseCase {}

class _CloseLocalConfigStorageUseCaseMock extends Mock
    implements CloseLocalConfigStorageUseCase {}
```
To make it more clear what this is about.